runtime/complex.go

```diff
@@ -49,6 +49,7 @@ func (c *Complex) Value() complex128 {
 	return c.value
 }
 
+// TODO: Move this function below the initComplexType()
```

Comment:
Sorry! I gave you the wrong advice, slot and method functions should be above the initXXX() function and helpers (e.g. complexArithmeticOp) should be below.